tests/integration/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "node-esm-harness",
+  "version": "1.0.0",
+  "description": "A test harness for testing the ar-io-sdk in ESM project'",
+  "type": "module",
+  "scripts": {
+    "postinstall": "yarn link @ar.io/sdk && yarn update-aos-ant",
+    "update-aos-ant": "node update-aos-ant-wasm.mjs",
+    "test": "node --import=../../register.mjs --trace-warnings --test index.test.ts && echo '\nNode ESM Integration ✅'"
+  },
+  "dependencies": {
+    "@ar.io/sdk": "*",
+    "@permaweb/aoconnect": "^0.0.59",
+    "arweave": "^1.15.5"
```

Comment:
these can all be dev deps with this structure, no?

Reply:
[129ba8d](https://github.com/ar-io/ar-io-sdk/pull/387/commits/129ba8d470f411282778ae8acac043716297d45b)